### DISCUSSION TITLE
4K video playback is not working (enabled AV1).

### DISCUSSION THREAD

**Discussion by letsconfuse** [FreeTubeApp/FreeTube#4382] [General]
I just downloaded FreeTube today. I don't know much about it, but I am using uYouTube on my iPhone, so I know that enabling the AV1 format will give 4K playback, but after that, it is not working in this video. I have watched other 4K videos after enabling this AV1.

![Screenshot 2023-11-25 184134](https://github.com/FreeTubeApp/FreeTube/assets/152010164/af43a21c-9106-4949-a511-fb954a52d639)

![Screenshot 2023-11-25 184427](https://github.com/FreeTubeApp/FreeTube/assets/152010164/ef256cf1-088e-4352-bec1-9bc18995cf99)

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
duplicate of https://github.com/FreeTubeApp/FreeTube/issues/2013
